testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterViewControllerTest.m

```diff
@@ -6,6 +6,16 @@
 #import <XCTest/XCTest.h>
 #import "AppDelegate.h"
 
+@interface FlutterViewControllerFirstFrameOverride : FlutterViewController
+@property(nonatomic) BOOL firstFrameRendered;
+@end
+
+@implementation FlutterViewControllerFirstFrameOverride
+- (void)firstFrameDidRender {
```

Comment:
And firstFrameDidRender would go away.

Reply:
\o/

That's my general feedback too. Either make it actually functional in production like on Android or make it test only and private.